docs/deploy.md

```diff
@@ -199,6 +194,13 @@ specifying the ingress domain information like so in the `ingress.hosts` setting
           pathType: ImplementationSpecific
 ```
 
+##### SSL
+
+If you wish to put your Kubernetes Sematic deployment behind SSL, the recommended way to do
+this is to set up an [ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/)
+that points to the service (named `sematic-server`) deployed by the Helm chart,
```

Comment:
Don't we need an ingress for socketio in order to run the CloudResolver in attached mode?

Reply:
technically no?  i think i'm being nitpicky, but it's theoretically possible that someone just puts up an internal-only service in front of the two _without_ an external ingress, and then uses that for in-cluster-only private communication.  unlikely though, so i'm indifferent how to word this.  suggestions welcome.

Reply:
I was referring to a workflow when the user runs the Resolver on their own machine instead of in the driver pod, but that's pretty niche. I'm fine with not documenting it.